io/plugins/eu.esdihumboldt.hale.io.gml/src/eu/esdihumboldt/hale/io/gml/writer/internal/StreamGmlWriter.java

```diff
@@ -1752,6 +1785,8 @@ private void writeElement(Object value, PropertyDefinition propDef, IOReporter r
 			boolean hasValue = propDef.getPropertyType().getConstraint(HasValueFlag.class)
 					.isEnabled();
 
+			boolean isIdentifier = propDef.getDisplayName().equals("identifier");
```

Comment:
For identifying if this is the correct property, the local part and the namespace should be checked.
The `StreamGmlWriter` already has a property you can use to identify the current GML namespace: https://github.com/halestudio/hale/blob/1a25d6c351035ec4c3e162fdf7db4dcc1ee4d07c/io/plugins/eu.esdihumboldt.hale.io.gml/src/eu/esdihumboldt/hale/io/gml/writer/internal/StreamGmlWriter.java#L260